src/html/banner-display.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Initialises the plugin requriements.
+ *
+ * @package ShutterstockAudioWidget
+ */
+
+
+
+namespace SeagynDavis\SouthAfricaCovid19Banner\BannerDisplay;
+
+add_filter( 'body_class', function( $classes ) {
+
+  $options = get_option( 'rbd_options' );
+
+  return array_merge( $classes, array( $options[ 'rbd_position' ] ) );
+
+} );
+
+function header_banner(){
+
+  $top_banner_array = array( 'banner_topScroll', 'banner_topFixed' );
+  $options = get_option( 'rbd_options' );
+
+  if ( !in_array( $options[ 'rbd_position' ], $top_banner_array ) ){
+    return;
+  }
+
+  banner();
+}
+
+function footer_banner(){
+
+  $bottom_banner_array = array( 'banner_bottomScroll', 'banner_bottomFixed' );
+  $options = get_option( 'rbd_options' );
+
+  if ( !in_array( $options[ 'rbd_position' ], $bottom_banner_array ) ){
+    return;
+  }
+
+  banner();
+}
+
+function banner(){
```

Comment:
Just picked this up now. In here, there should only be HTML. So if you can use this for HTML (like in mine) and then have these functions in src (like I have for widget/shortcode).